config/routes.rb

```diff
@@ -19,73 +19,64 @@
 Rails.application.routes.draw do
   get '/health_check', to: 'health_check#all'
   get '/healthz', to: 'health_check#all'
-  root 'main#index'
+  root 'application#index'
 
   # A monkey patch for supporting links coming from legacy LTI tools that hardcoded the launch under tool.
-  scope ENV['RELATIVE_URL_ROOT_LEGACY'] || 'lti' do
-    get '(:tenant)/tool(.xml)', to: 'tool_profile#xml_config_legacy', app: ENV['DEFAULT_LTI_TOOL'] || 'default'
-    post '(:tenant)/tool', to: 'message#basic_lti_launch_request_legacy', as: 'blti_launch_legacy', app: ENV['DEFAULT_LTI_TOOL'] || 'default'
-  end
-
-  scope ENV['RELATIVE_URL_ROOT'] || 'lti' do
-    get '/health_check', to: 'health_check#all'
-    get '/healthz', to: 'health_check#all'
-
-    # rooms calls this api to validate launch from broker
-    namespace :api do
-      namespace :v1 do
-        get 'users/:id', to: 'users#show', as: :users
-        get 'user', to: 'users#show', as: :user
-        get 'sessions/:token', to: 'sessions#validate_launch', as: :sessions
-      end
-    end
-
-    # grades
-    get 'grades/:grades_token/list', to: 'grades#grades_list', as: :grades_list
-    post 'grades/:grades_token/change', to: 'grades#send_grades', as: :send_grades
-
-    # registration (LMS -> broker)
-    get 'registration/list', to: 'registration#list', as: :registration_list
-    get 'registration/new', to: 'registration#new', as: :new_registration
-    get 'registration/edit', to: 'registration#edit', as: :edit_registration
-    post 'registration/submit', to: 'registration#submit', as: :submit_registration
-    get 'registration/delete', to: 'registration#delete', as: :delete_registration
-
-    # registration (broker -> rooms)
-    use_doorkeeper do
-      # Including 'skip_controllers :application' disables the controller for managing external applications
-      #   [http://example.com/lti/oauth/applications]
-      skip_controllers :applications unless ENV['DEVELOPER_MODE_ENABLED'] == 'true'
+  get '(:tenant)/tool(.xml)', to: 'tool_profile#xml_config_legacy', app: ENV['DEFAULT_LTI_TOOL'] || 'default'
```

Comment:
Be advised about the scoping of the legacy endpoints to `RELATIVE_URL_ROOT` and not `RELATIVE_URL_ROOT_LEGACY` since with this PR in such case admins will have legacy endpoint under `/$RELATIVE_URL_ROOT/$RELATIVE_URL_ROOT_LEGACY`